modules/custom/az_publication/config/install/az_publication.az_citation_style.az_ama.yml

```diff
@@ -3,4 +3,5 @@ status: true
+style: american-medical-association
+custom: 0
```

Comment:
Traditionally, we've used the boolean type
See https://github.com/az-digital/az_quickstart/blob/main/themes/custom/az_barrio/config/schema/az_barrio.schema.yml#L160-L162
See https://github.com/az-digital/az_quickstart/blob/eb2f6418c61f6c3dcf8d9bfb6b3ef250e7697a2c/themes/custom/az_barrio/config/install/az_barrio.settings.yml#L2